src/js/control-bar/progress-control/mouse-time-display.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file mouse-time-display.js
+ */
+import Component from '../../component.js';
+import MouseDisplayBar from './mouse-display-bar.js';
+import SeekBar from './seek-bar.js';
+import * as Fn from '../../utils/fn.js';
+import formatTime from '../../utils/format-time.js';
+import throttle from 'lodash-compat/function/throttle';
+
+/**
+ * The Mouse Time Display component shows the time you will seek to
+ * when hovering over the progress bar
+ *
+ * @param {Player|Object} player
+ * @param {Object=} options
+ * @extends Component
+ * @class MouseTimeDisplay
+ */
+class MouseTimeDisplay extends SeekBar {
+
+  constructor(player, options) {
+    super(player, options);
+    this.update();
+    player.on('ready', () => {
+      this.on(player.controlBar.progressControl.el(), 'mousemove', throttle(Fn.bind(this, this.handleMouseMove), 50));
```

Comment:
Yeah that'd be cool. There's `dom.findElPosition` which does some of it, but I didn't use it because I don't think it's exported. But something more like a getRelativePosition dom function would be useful.